app/js/components/filterComponents/totalRiskSelect/totalRiskSelect.controller.js

```diff
@@ -53,18 +59,27 @@ function totalRiskSelectCtrl($location,
             });
 
         $scope.label = $scope.selected.label;
-
+        setSeverity($scope.selected.value);
```

Comment:
setSeverity is being set twice in this function. Also, you are using option.value and $scope.selected.value. Since $scope.selected is being set you can probably get rid of the option param.